lib/ansible/module_utils/urls.py

```diff
@@ -98,14 +98,24 @@
     # Python 3
     import http.client as httplib
 
+import ansible.module_utils.six.moves.urllib.request as six_request
+from ansible.module_utils.six.moves.urllib.request import (build_opener,
+        install_opener, urlopen, BaseHandler, HTTPBasicAuthHandler,
+        HTTPPasswordMgrWithDefaultRealm, HTTPRedirectHandler,
+        ProxyHandler, Request)
```

Comment:
My preference here would be to just import request as six_request and remove the object imports.
Mostly because of:

* It's a lot of imports, and a lot of imports on a single line

* In this case, it would be useful to refer to this objects via module scope, since they are not (always) the regular urlib/urllib2 objects. Using the six_request module namespace makes that more clear.

But, thats a style thing, so does not require revision now.